python-frontend/pom.xml

```diff
@@ -37,6 +37,68 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>${protobuf.version}</version>
+    </dependency>
   </dependencies>
-
+  <build>
```

Comment:
Just to make sure, this is the same thing we do in sonar-security to compile the Java Protobuf classes automatically, right?
Will it need adaptations to build the Python classes as well? (Should we even do it here as well?)

Reply:
Yes, that's the same as sonar-security. I believe it makes sense to also build the Python classes here, either on this pom or in a dedicated pom if we decide to have a separate module